fda-model/alternatives/Alternative.cs

```diff
@@ -178,7 +187,9 @@ private static ConsequenceDistributionResult IterateOnAAEQ(ConsequenceDistributi
                 if (_completedIterations % progressChunks == 0)//need an atomic integer count here.
                 {
                     double percentcomplete = _completedIterations / (double)_ExpectedIterations * 100;
-                    aaeqResult.ReportProgress(aaeqResult, new ProgressReportEventArgs((int)percentcomplete));
+                    //TODO: We need to refactor the way that progress is being reported 
```

Comment:
Again, i don't really know what this means, just calling attention to it.

Reply:
same idea - there are additional dimensions we need to figure out